src/Typography/FontFactory.php

```diff
@@ -36,6 +36,20 @@ public function file(string $value): self
         return $this->filename($value);
     }
 
+    public function strokeColor(mixed $value): self
+    {
+        $this->font->setStrokeColor($value);
+
+        return $this;
+    }
+
+    public function strokeWidth( ?int $value): self
+    {
+        $this->font->setStrokeWidth($value);
+
+        return $this;
+    }
+
```

Comment:
Let's combine and replace this with one method: `stroke(int $width, mixed $color)` in `FontFactory`.

Reply:
makes sense.